README.md

```diff
@@ -1,16 +1,42 @@
+COSI repos and images:
+
+[Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec)  \
+[API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) \
+[Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) <br/>
+&emsp; - [images: cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) \
```

Comment:
Why are we using html tags when markdown provides rich formatting

Reply:
It was part of the effort to get the table to work. I will update them to backslashes on the next PR mentioned below.

Reply:
I can update in this pr actually and make new pr's for the other 4 README.md.